action-server/covidflow/actions/test_navigation_form.py

```diff
@@ -85,7 +85,8 @@ async def validate_test_navigation__postal_code(
 
             testing_locations = await get_testing_locations(coordinates)
 
-        except:
+        except Exception as error:
+            logger.exception(error)
```

Comment:
I think it should simply be:
```suggestion
        except Exception:
            logger.exception("Failed to fetch testing locations")
```
No need for the exception as argument the trace should be attached to the log entry. 

@mathieubergeron can you confirm?

Reply:
yes :)